README.md

```diff
@@ -24,21 +25,20 @@
 8. [Places to Learn More About DSC](#places-to-learn-more-about-dsc)
 9. [License](#license)
 
-## Module Description
+## Description
 
-Puppet module for managing Windows PowerShell DSC (Desired State Configuration) resources.
+The Puppet dsc module manages Windows PowerShell DSC (Desired State Configuration) resources.
```

Comment:
Names of modules are lowercased by convention. It helps differentiate them from the software they support.